doc/PLUGINS.md

```diff
@@ -200,6 +200,20 @@ corresponding payloads are listed below.
 
 ### Notification Types
 
+#### `channel_opened`
+
+A notification for topic `channel_opened` is sent if a peer successfully funded a channel
+with us. It contains the peer id, the funding amount, and the feerate (per kw).
+```
+{
+	"channel_opened": {
+		"id": "03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3f8f",
+		"funding_satoshis": "100000000msat",
+		"feerate": 7500,
```

Comment:
`feerate` seems like an odd detail to add here, but something like the funding txid would be useful here imho.

Reply:
Ok, I'll replace the feerate by the txid